lib/capistrano/tasks/deploy.rake

```diff
@@ -16,7 +16,9 @@ namespace :deploy do
       invoke 'magento:setup:di:compile'
       invoke 'magento:setup:permissions'
       within current_path do
-        execute :magento, 'maintenance:enable'
+        if test 'ls bin/magento'
```

Comment:
@davidalger You may want to encapsulate this test inside of its own discrete task like `magento:is-installed` or something like that, as I could foresee this test needing to be reused in the future. If you'd like me to refactor this code, let me know.